scss/mixins/_utilities.scss

```diff
@@ -1,6 +1,20 @@
 // Utility generator
 // Used to generate utilities & print utilities
-@mixin generate-utility($utility, $infix, $is-rfs-media-query: false) {
+@mixin generate-utility($utility, $utility-name, $infix, $is-rfs-media-query: false) {
```

Comment:
We don't need this new argument (which would lead to a breaking change) since we can get the field using `$properties: map-get($utility, property)` and `nth($properties, 1)` as it's done L20, isn't it?

Reply:
The problem is that we are already passing the values, but not the name of the utility. `nth ($properties, 1)` can return multiple values or nothing at all if the property field does not exist. 

I can move the name of the utility to the end of the argument chain and make it an optional field. Then I'll add an extra check for the existence of this argument. This will not break the ready-made code for developers. 

And you can apply by name. For example, like this: `@include generate-utility ($ utility, $ infix, $ utility-name: $ key); `